core/common/src/main/java/alluxio/metrics/sink/CsvSink.java

```diff
@@ -38,6 +43,7 @@
 
   private CsvReporter mReporter;
   private Properties mProperties;
+  private File mFile;
```

Comment:
- `final` ? put `final` whenever possible
- should be `mDir` as this indicates poll dir?